components/infobox/commons/infobox_person.lua

```diff
@@ -289,4 +307,39 @@ function Person:_createTeam(team, link)
 	return '[[' .. link .. '|' .. team .. ']]'
 end
 
+--- Allows for overriding this functionality
+function Person:getCategories(args, birthDisplay, personType, status)
+	local categories = { personType .. 's' }
+
+	if not args.teamlink and not args.team then
+		table.insert(categories, 'Teamless ' .. personType .. 's')
+	end
+	if args.country2 or args.nationality2 then
+		table.insert(categories, 'Dual Citizenship ' .. personType .. 's')
+	end
+	if args.death_date then
+		table.insert(categories, 'Deceased ' .. personType .. 's')
+	end
+	if
+		args.retired == 'yes' or args.retired == 'true'
+		or string.lower(status or '') == 'retired'
+		or string.match(args.retired or '', '%d%d%d%d%')--if retired has year set apply the retired category
+	then
+		table.insert(categories, 'Retired ' .. personType .. 's')
+	else
+		table.insert(categories, 'Active ' .. personType .. 's')
+	end
+	if not args.id then
+		table.insert(categories, 'InfoboxIncomplete.')
```

Comment:
```suggestion
		table.insert(categories, 'Incomplete Infoboxes')
```

Reply:
Although maybe this should just trigger an error way earlier.

Reply:
it is the name of the exiting category used on all wikis i checked (minus the `.` which is a typo)

Reply:
Is the name actually used for anything though?

Reply:
that idk, i would suggest no, maybe some wikis have it set as `__HIDDENCAT__`, but then they could just make the new one the same way

i will check later today how many pages are in that cat on the main wikis, then we can decide tomorrow if we want to just let it error or not

Reply:
* heroes 2
* hearthstone 5+ 1 user
* sc 4
* r6 1
* apex 2
* pubg 1
* dota2 1
* sc2 0+ 2 user

so imo we can just let it error
will update it tomorrow accordingly

Reply:
Cool 👍